batch7crud-roll4/src/main/webapp/resources/js/PostForm.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Created by pratishshr on 2/1/16.
+ */
+
+function PostForm() {
+    var deleteBtn = document.getElementsByClassName('delete-btn');
+    var logoutBtn = document.getElementsByClassName('logout-btn')[0];
+
+    var that = this;
+
+    this.init = function () {
+        for (var i = 0; i < deleteBtn.length; i++) {
+            deleteBtn[i].onclick = function (e) {
+                e.preventDefault();
+                var href = this.href;
+                var confirmation = confirm('do you want to delete?');
+
+                if (confirmation == true) {
+                    that.submitForm(href);
+                }
+            }
+        }
+
+        logoutBtn.onclick = function (e) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ASemicolon)